envoy.dependency.check/tests/test_abstract_release.py

```diff
@@ -111,7 +111,7 @@ async def test_release_release(patches, raises, err, provided):
     kwargs = {}
     if provided:
         kwargs["release"] = MagicMock()
-    release = DummyDependencyGithubRelease(repo, "VERSION", **kwargs)
+    release = DummyDependencyGithubRelease(repo, "VERSION", ["URL"], **kwargs)
```

Comment:
im trying to understand why these need to be changed - but i will look at tests more in next review